pkg/mgrconfig/load.go

```diff
@@ -337,8 +376,11 @@ func ParseEnabledSyscalls(target *prog.Target, enabled, disabled []string) ([]in
 			syscalls[call.ID] = true
 		}
 	}
+
 	for call := range syscalls {
-		if target.Syscalls[call].Attrs.Disabled {
+		if target.Syscalls[call].Attrs.Disabled ||
```

Comment:
This function needs to check that descriptionsMode != Invalid. Otherwise typos in the config  will be silently ignored.

Reply:
Done!